include/nlohmann/detail/input/json_sax.hpp

```diff
@@ -611,11 +619,11 @@ class json_sax_dom_callback_parser
     /// the parsed JSON value
     BasicJsonType& root;
     /// stack to model hierarchy of values
-    std::vector<BasicJsonType*> ref_stack;
+    std::vector<BasicJsonType*> ref_stack{};
     /// stack to manage which values to keep
-    std::vector<bool> keep_stack;
+    std::vector<bool> keep_stack{};
     /// stack to manage which object keys to keep
-    std::vector<bool> key_keep_stack;
+    std::vector<bool> key_keep_stack{};
```

Comment:
Bracket isn't necessary for types with default constrcutor like `std::string` and `std::vector`, correct? Is there an argument of consistency in initialization?